src/dapps/ManageENS/ManageENS.vue

```diff
@@ -354,7 +354,7 @@ export default {
       if (this.parsedTld !== '' && isSupported === undefined) {
         Toast.responseHandler(
           `Domain TLD ${this.parsedTld} is not supported in this node!`,
-          Toast.ERROR
+          Toast.ERROR``
```

Comment:
I don't think these tildes are necessary

Reply:
you are right:) definitely not necessary